packages/web3-common/package.json

```
@@ -0,0 +1,41 @@
+{
+	"name": "web3-common",
+	"version": "4.0.0-alpha.0",
```

```suggestion
	"version": "1.0.0-alpha.0",
```
Package currently doesn't exist, so v1 not v4